src/griml/__init__.py

```diff
@@ -0,0 +1,6 @@
+try:
+    from importlib.metadata import version
+except ImportError:  # For Python <3.8 compatibility
+    from importlib_metadata import version
```

Comment:
I guess there's no reason for the try/except statement since your package doesn't support version <3.8. So the second line should be all that you need.

Reply:
True! I can take these lines out.